src/aioftp/client.py

```diff
@@ -1003,25 +1179,25 @@ async def upload(self, source, destination="", *, write_into=False, block_size=D
         :param block_size: block size for transaction
         :type block_size: :py:class:`int`
         """
-        source = pathlib.Path(source)
-        destination = pathlib.PurePosixPath(destination)
+        source_ = pathlib.Path(source)
+        destination_ = pathlib.PurePosixPath(destination)
```

Comment:
To describe the wrapped context this should named as `destination_path` and `source_path`.

Reply:
Fixed